app/views/external_members/supervisions/_orientation.html.erb

```diff
@@ -1,23 +1,23 @@
 <tr>
-  <td class="text-center">
-    <div class="item-action dropdown">
-      <a data-toggle="dropdown" class="icon" aria-expanded="false"><i class="fe fe-more-horizontal"></i></a>
-      <div class="dropdown-menu dropdown-menu-right" x-placement="top-end">
+   <td>
+      <div class="dropdown">
+         <%= link_to orientation.academic.name, external_members_supervision_path(orientation), class: 'academic-name-link', 'data-toggle': 'dropdown' %>
```

Comment:
The class `class: 'academic-name-link'` is no more necessary. So can be deleted. See all files.

You need to change in all test not use this class, like we did in responsible orientation test.